pkgs/net-doc/net/scribblings/server.scrbl

```diff
@@ -249,10 +249,10 @@ process and does not rely on any networking machinery:
   (define ch (make-channel))
   (define stop
     (start-server
-     #:listen-proc (λ (port backlog reuse? host) ch)
      #:accept-proc (λ (ports) (apply values ports))
      #:close-proc void
-     "127.0.0.1" 0 echo))
+     ch
+     echo))
   (code:line)
```

Comment:
Looks like the docs for `run-server` also need to be updated to match the signature changes.